apps/shadcn-example/src/components/wallet-connection/connect-wallet.tsx

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
any reason we dont simply include it in the main demo dapp?

Reply:
I replied to your later comment on this